tests/core/test_integration.py

```diff
@@ -505,7 +505,7 @@ def test_hourly_model_with_lookback_no_backfill_in_dev(init_and_plan_context: t.
         ]
 
 
-@freeze_time("2023-01-08 00:00:00")
+@time_machine.travel("2023-01-08 00:00:00 UTC", tick=False)
```

Comment:
Why was tick required here but not other tests?

Reply:
In some tests, we only capture time at one point - so the result is the same whether or not time keeps going

Reply:
But if we capture time twice, we'll only get the same value if `tick=False`.

The default in freezegun was `tick=False`, but it's `True` in time-machine.